lib/modules/manager/pub/artifacts.spec.ts

```diff
@@ -19,7 +19,9 @@ process.env.CONTAINERBASE = 'true';
 const lockFile = 'pubspec.lock';
 const oldLockFileContent = 'Old pubspec.lock';
 const newLockFileContent = 'New pubspec.lock';
-const depName = 'depName';
+const depNames = Array.from({ length: 3 }, (_, idx) => `depName${idx}`);
```

Comment:
please use our range function to create an array of specific numbers, search code for sample